schemas/session-replay/full-snapshot-record-schema.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "full-snapshot-record-schema.json",
+    "title": "FullSnapshotRecord",
+    "description": "Schema of a Record type which contains the full snapshot of a screen.",
+    "tsType": "BrowserFullSnapshotRecord | MobileFullSnapshotRecord"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏This will only work for TS right? Doesn't `oneOf` with some `$ref` work?

Reply:
While it was generating duplicate types before, now that we've 1) fixed the problem Android generator introduced by requiring a `type` definition next to a `$ref` and 2) made completely isolated mobile & browser schemas, it works just fine.

Pushed this change in a commit.